ui/src/app/pages/cookie-policy.tsx

```diff
@@ -46,7 +47,11 @@ const COOKIE_DELETION_LINK = 'https://www.aboutcookies.org/how-to-delete-cookies
 const GOOGLE_PRIVACY_LINK = 'https://policies.google.com/privacy';
 const ZENDESK_PRIVACY_LINK = 'https://www.zendesk.com/company/customers-partners/privacy-policy/';
 
-export class CookiePolicyComponent extends React.Component<{}, {}> {
+export class CookiePolicy extends React.Component<{}, {}> {
+  componentDidMount() {
```

Comment:
Please take a look at [this document](https://docs.google.com/document/d/11wJvrVUkgkOxwaHsHwg0JD3oqYunRAnLxDwI55INXYA/edit#) about how I could do this better (but not in this PR)